build-ci/new_package.rst

```diff
@@ -2,24 +2,35 @@
 Adding a New Package to the Build
 #################################
 
-"Top-level products" are the roots of the LSST package hierarchy. They include
-``lsst_apps``, ``lsst_distrib``, ``qserv_distrib`` and ``lsst_sims``. Before
-adding a new dependency to any top level product, it must be added to the
-`LSST organization on GitHub`_ and access must be granted to the appropriate
-teams. Both DM-written packages and third-party packages should go through the
-RFC process first to confirm their name and the suitability of their contents.
-
-DM-written Science Pipelines packages should follow the template in the
-`lsst/templates`_ repository.
-
-Third-party packages should be packaged as described in :doc:`third_party`.
+New packages intended for distribution to end users should generally be added
+as a dependency of a "top level products": these are the roots of the LSST
+package hierarchy. They include ``lsst_apps``, ``lsst_distrib``,
```

Comment:
although `lsst_distrib` contains `lsst_apps` so it's hard to say `lsst_apps` is a root... We treat it as a root though.